src/iop/colorbalancergb.c

```diff
@@ -348,6 +348,25 @@ int legacy_params(dt_iop_module_t *self, const void *const old_params, const int
   return 1;
 }
 
+void init_presets(dt_iop_module_so_t *self)
+{
+  // Note : all the elements of the params structure are scalar floats,
+  // so we can just init them all to 0.f in batch
+  // Then, only 4 params have to be manually inited to non-zero values
+  dt_iop_colorbalancergb_params_t p = { 0.f };
+  p.shadows_weight = 1.f;        // $MIN:  0.0 $MAX:   3.0 $DEFAULT: 1.0 $DESCRIPTION: "shadows fall-off"
+  p.highlights_weight = 1.f;     // $MIN:  0.0 $MAX:   3.0 $DEFAULT: 1.0 $DESCRIPTION: "highlights fall-off"
+  p.mask_grey_fulcrum = 0.1845f; // $MIN:  0.0 $MAX: 1.0 $DEFAULT: 0.1845 $DESCRIPTION: "mask middle-gray fulcrum"
+  p.grey_fulcrum = 0.1845f;      // $MIN:  0.0 $MAX: 1.0 $DEFAULT: 0.1845 $DESCRIPTION: "contrast gray fulcrum"
```

Comment:
do you still want the introspection stuff here?

Reply:
as a memo, it's useful, yes. The introspection is not supposed to pick it here, is it ?

Reply:
Better to not have introspection stuff lying around, since introspection gets picked as a perl script analysing files so it might throw off stuff... (haven't tried though)

Reply:
Feel free to add comments that don't look like introspection though.